src/py/flwr/cli/app.py

```diff
@@ -18,6 +18,7 @@
 
 from .example import example
 from .new import new
+from .run import run
 
 app = typer.Typer(
```

Comment:
Is or will there be a (for instance) `--help` flag for instance which prints an example usage?